src/promptflow-evals/promptflow/evals/evaluate/_utils.py

```diff
@@ -214,3 +214,22 @@ def _apply_column_mapping(source_df: pd.DataFrame, mapping_config: dict, inplace
 
 def _has_aggregator(evaluator):
     return hasattr(evaluator, "__aggregate__")
+
+
+def _get_trace_destination_config(project_scope: str = None):
+    from promptflow._sdk._configuration import Configuration
+
+    trace_destination = None
+
+    if project_scope is not None:
+        trace_destination = _trace_destination_from_project_scope(project_scope)
+
+    pf_config = Configuration(overrides={
+        "trace.destination": trace_destination} if trace_destination is not None else None)

```

Comment:
NIT: Do you need 'if trace_destination is not None else None'?

Reply:
However we cannot validate it since tracing will happen only when cosmos db is initialized already otherwise it needs to be initialized. It is better to fail here than at time of uploading the run

Reply:
When we initialze PFClient(config={"trace.destination": trace_destination}), the configuration check will happen again. 
Will PFClient() load the destination from local config if no config overriable pass to the constrcutor? If so, maybe that's a better place for this change.

Reply:
Maybe we can call `pf_client._config.get_config(key)` to retrieve the config the back after the PFClient initialization?

Reply:
Thanks for the suggestion. Made the change.